steps/sofy-upload-bitrise-step/0.0.2/step.yml

```
@@ -0,0 +1,69 @@
+title: Sofy-app-upload-bitrise-step
```

```suggestion
title: Sofy app upload
```

Title is usually a human-readable string that is displayed on the UI. Feel free to adjust my suggestion